src/components/dashboard/GapByRace.vue

```diff
@@ -1,26 +1,74 @@
 <template>
-  <div class="gap-container">
-    <div>
-      <GapChart
-        :active-stats="activeStats"
-        :expected="expected"
-        :field-data="fieldData"
-      />
+  <div class="outer-container">
+    <div
+      class="gap-container"
+      :class="{ visible: focusStat.value === 'total' }"
+    >
+      <div>
+        <GapChart
+          :active-stats="activeStats"
+          :expected="expected"
+          :field-data="fieldData"
+        />
+      </div>
+      <div class="centered">
+        <p class="has-text-centered">
+          In {{ activeCluster.name }}, the largest gap is among
+          {{ minVaxRace?.name }} residents. Only
+          <strong
+            >{{ formatPct(minVaxRace?.pct) }} of
+            {{ minVaxRace?.name }} residents</strong
+          >
+          are vaccinated compared to {{ formatPct(expected) }} statewide.
+          <strong
+            >{{ minVaxRace?.gap }} more {{ minVaxRace?.name }} residents</strong
+          >
+          need to be vaccinated to close this gap.
+        </p>
+      </div>
     </div>
-    <div class="centered">
-      <p class="has-text-centered">
-        In {{ activeCluster.name }}, the largest gap is among
-        {{ minVaxRace?.name }} residents. Only
-        <strong
-          >{{ formatPct(minVaxRace?.pct) }} of
-          {{ minVaxRace?.name }} residents</strong
-        >
-        are vaccinated compared to {{ formatPct(expected) }} statewide.
-        <strong
-          >{{ minVaxRace?.gap }} more {{ minVaxRace?.name }} residents</strong
-        >
-        need to be vaccinated to close this gap.
-      </p>
+    <div
+      class="gap-container"
+      :class="{
+        visible:
+          focusStat.value !== 'total' && activeFocusStats?.population > 0,
+      }"
+    >
+      <div
+        class="is-flex is-flex-direction-row is-justify-content-space-around"
+      >
+        <KeyPerformanceIndicator
+          :value="formatPct(activeFocusStats?.pct)"
+          title="Vaccinated in Community"
+        />
+        <KeyPerformanceIndicator
+          :value="activeFocusStats?.gap ?? NaN"
+          title="Doses to Close Gap"
+        />
+      </div>
+      <div class="centered">
+        <p class="has-text-centered">
+          In {{ activeCluster.name }},
+          <strong>{{ formatPct(activeFocusStats?.pct) }}</strong> of
+          {{ activeFocusStats?.name }} residents are are vaccinated compared to
+          our goal of {{ formatPct(expected) }} total vaccinations statewide.
+          <strong
+            >{{ activeFocusStats?.gap }} more
+            {{ activeFocusStats?.name }} residents</strong
+          >
```

Comment:
if the gap is 0, should this text change?

Reply:
agreed, in the case below, maybe the text should report the stats on the Asian gap and/or state that there is not enough data on vaccination rates for Black residents of the selected area.

Reply:
I've updated this wording !

Reply:
Hmm, working on this !

Reply:
<img width="1060" alt="Screen Shot 2022-05-03 at 10 46 33 AM" src="https://user-images.githubusercontent.com/5270855/166486485-6910eecf-3cc8-4960-8dc2-ffc92daf325d.png">
 Updated to this ~